lib/simplefootball_web_web/views/competition_view.ex

```diff
@@ -0,0 +1,16 @@
+defmodule SimplefootballWebWeb.CompetitionView do
+  use SimplefootballWebWeb, :view
+
+  def renderList(competitions) do
+    Enum.map(competitions, fn competition -> renderCompetition(competition) end)
+  end
+
+  def renderCompetition(competition) do
+    %{
+      name: competition.name,
+      iconUrl: competition.icon_url,
+      id: competition.id,
```

Comment:
Es ist besser, wenn man die Datenbank IDs rein als internes Implementierungsdetail betrachtet.

Die *Identität* einer Entität ist hingegen domänenspezifisch, hier könnte man zB einfach den `competitionType` nehmen, weil der ja auch unique sein sollte.
Dafür könnte man `competitionType` irgendwie umbenennen, um der Tatsache gerecht zu werden, dass es eigentlich ein unique identifier.

Zu dem Thema ist in "Implementing Domain Driven Design" das Kapitel "Unique Identity" interessant. Ab Seite 173 hier: https://drive.google.com/drive/u/0/folders/0B-9R2yIAk_haQzVJcXJYdE1FQjg